src/persist/Cargo.toml

```diff
@@ -20,6 +20,7 @@ abomonation = "0.7"
 abomonation_derive = "0.5"
 crossbeam-channel = "0.5"
 differential-dataflow = { git = "https://github.com/TimelyDataflow/differential-dataflow.git" }
+itertools = "0.10.1"
```

Comment:
What is this dep getting us? It's pretty lightweight, but is it really necessary to add?

Reply:
removed this dep in favor of using `consolidate_updates`